sycl/doc/extensions/proposed/sycl_ext_oneapi_syclbin.asciidoc

```diff
@@ -242,3 +242,142 @@ _{endnote}_]
 
 |====
 
+=== New free function for linking
```

Comment:
I think it would be better to add these overloads of `link` to the sycl_ext_oneapi_kernel_compiler instead.  That way, all the new overloads of `build`, `compile`, and `link` taking a `PropertyList` will be together in one extension.  Do you see a downside to that?

Reply:
I was originally planning on doing so, but the free functions introduced in ext_oneapi_kernel_compiler all take source-based kernel bundles. Adding the general link function there feels out-of-place.